### PR TITLE
Migrate repo URLs

### PR DESCRIPTION
**Pull request by yangshun**
Migrate some old repo URLs to the new repo URLs.

### CODE REVIEW

File: content/docs/codebase-overview.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The [fbjs repository](https://github.com/facebook/fbjs) exists because React sha
 After cloning the [React repository](https://github.com/facebook/react), you will see a few top-level folders in it:
 
 * [`src`](https://github.com/facebook/react/tree/master/src) is the source code of React. **If your change is related to the code, `src` is where you'll spend most of your time.**
-* [`docs`](https://github.com/facebook/react/tree/master/docs) is the React documentation website. When you change APIs, make sure to update the relevant Markdown files.
+* [`docs`](https://github.com/reactjs/reactjs.org/tree/master/content) is the React documentation website. When you change APIs, make sure to update the relevant Markdown files.
```
Comment (yangshun) [inline content/docs/codebase-overview.md]: Strictly speaking this line should be removed because it the docs is no longer part of the facebook/react repo.

Comment (bvaughn) [reply content/docs/codebase-overview.md]: I think it's actually pretty important to keep this link here. 😄

Comment (gaearon) [reply content/docs/codebase-overview.md]: We should make a pass after repo re-organization and rewrite this.

Comment (bvaughn) [reply content/docs/codebase-overview.md]: Definitely.

Comment (yangshun) [reply content/docs/codebase-overview.md]: Sorry I meant that it should be removed from this section and rephrased to suit the new repo structure. Definitely agree we should keep the link in this page!

**Pull request merged**